packages/core/demo/index.html

```diff
@@ -17,12 +17,16 @@
 				<h1>Carbon Charts</h1>
 				<h3>Basic chart types: Demo</h3>
 
-				<a class="bx--btn bx--btn--primary" href="https://github.com/IBM/carbon-charts">
-					Project Repository
+				<a class="bx--btn bx--btn--primary" href="https://github.com/IBM/carbon-charts" target="_blank">
+					Repository
 				</a>
 
-				<a class="bx--btn bx--btn--primary" href="https://pages.github.com/IBM/carbon-charts/documentation/">
-					Documentation
+				<a class="bx--btn bx--btn--primary" href="https://charts.carbondesignsystem.com/documentation/" target="_blank">
+					Docs
```

Comment:
What's wrong with full words?

Reply:
I think docs sufficiently explains where the link is pointing to, also it's the *slang* most developers use for documenatations